elements/body.html

```diff
@@ -1,20 +1,14 @@
 <article class="article">
   <header class="article-header">
     <h2>
-      <code>
-        <!-- Actual Tag Name  -->
-        &lt;body>
-      </code>
-      <!-- Audio -->
+      <code>&lt;body&gt;</code>
       <audio src="audio/body.mp3" controls></audio>
```

Comment:
I noticed you removed audio from several other elements.  What's the reason for leaving it in for this one?